teams/vsa/onboarding/VSA Technical Orientation.md

```diff
@@ -1,354 +1,489 @@
+| Chrome&nbsp;Extension: _Proxy&nbsp;SwitchyOmega_ | [Proxy SwitchyOmega](https://chrome.google.com/webstore/detail/proxy-switchyomega/padekgcemlokbadohgkifijomclgjgif)                                            | Needed to access internal DevOps web sites sites--[such as Jenkins, Sentry, Prometheus, Grafana](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/internal-tools.md)--over SOCKS5 proxy.                                                                                                                                                   |
+| Chrome&nbsp;Extension: _axe_                     | [axe&nbsp;-&nbsp;Web&nbsp;Accessibility Testing](https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd)         | [508 accessibility scanning](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-accessibility-best-practices.md#getting-started-with-automation): _required_. Catch accessibility issues during development _before_ they are flagged by the build.                                                                                    |
+| Chrome&nbsp;Extension: _axe-coconut_             | [axe-coconut&nbsp;-&nbsp;Web&nbsp;Accessibility Testing](https://chrome.google.com/webstore/detail/axe-coconut-web-accessibi/iobddmbdndbbbfjopjdgadphaoihpojp) | [508 accessibility scanning](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-accessibility-best-practices.md#getting-started-with-automation): _required for new development_. Catch WCAG 2.1 accessibility issues during development _before_ they are flagged by the build.                                                       |
+| Chrome&nbsp;Extension: _Lighthouse_              | [Lighthouse](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk)                                                            | [508 accessibility scanning](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-accessibility-best-practices.md#getting-started-with-automation): _recommended_. Audits for web performance, accessibility, progressive web apps, SEO and more.                                                                                        |
+
+### (optional) VS Code Extensions
```

Comment:
glad you added this, was on my to-do list. I ran a survey before and see that you've captured the big ones but also good ones we missed